test/app_test_case.py

```diff
@@ -65,14 +68,19 @@ def lambda_name(cls) -> str:
         """
         raise NotImplementedError
 
-    @property
-    def base_url(self):
+    def base_url(self, endpoint_path=None, **params) -> str:
```

Comment:
If we are going to lose this property and add an endpoint path as a parameter, why don't we use that functionality in the call sites?

IIRC we want to use furl to prevent unhappy accidents in URL composition from url encoding. Most of the call sites are not updated to use it though, they perform concatenation on the URL.

get(f'{self.base_url()}/awesome endpoint') should be avoided because we wont get a valid URL

self.base_url(endpoint='awesome endpoint') would work because the space is encoded.

Reply:
Yea, I had initially thought about that but doing it in this PR seemed excessive due to the many call sites, I opted for a FIXME approach to address it. However, the base PR was reverted to use the `@property` again, this is no longer a concern (I believe) but changes to URL generation using `f'` interpolations are [imminent](https://github.com/DataBiosphere/azul/issues/3008).